ui/components/ErrorBoundary.js

```diff
@@ -0,0 +1,40 @@
+import { Button } from "@material-ui/core";
+import React from "react";
+
+/**
+ * ErrorBoundary is a React component that catches JavaScript errors in its child components and renders a fallback UI when an error occurs.
```

Comment:
Same one used in `meshery-extensions`?

Reply:
This component usage will take affect on the Meshery-extensions as well.
With your comment, I just saw, that meshery-extensions is also using an errorBoundary class. Although its not same. It is using this package: https://www.npmjs.com/package/react-error-boundary

The behaviour is same though. I can just use that as well